test/trigger_cellranger_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# note, this test must be run from the workflow root for the relative paths to work out.
+# WDL file should be the KCO cellranger, located here:
+# https://portal.firecloud.org/#methods/single-cell-portal/cell-ranger-2-0-2/3/wdl
+
+set -e
+
+export CROMWELL_SECRETS=~/.ssh/mint_cromwell_config.json
```

Comment:
What is the structure of the `mint_cromwell_config.json`?

Reply:
Also it would be useful to allow users to pass in a path to CROMWELL_SECRETS, and use `~/.ssh/mint_cromwell_config.json` as a default